docs/products/postgresql/howto/migrate-db-to-aiven-via-console.rst

```diff
@@ -0,0 +1,278 @@
+Migrate PostgreSQL® databases to Aiven using the console 
+========================================================
+
+You can migrate PostgreSQL® databases to Aiven using either CLI (``aiven-db-migrate``) or the Aiven web console. This article addresses the latter scenario by providing guidelines on how to use the Aiven console to migrate PostgreSQL databases to the Aiven platform.
+
+.. seealso::
+
+    For the other migration method (using ``aiven-db-migrate``), see :doc:`Migrate to Aiven for PostgreSQL® with aiven-db-migrate </docs/products/postgresql/howto/migrate-aiven-db-migrate>`.
+
+About migrating via console
+---------------------------
+
+* With the console migration tool, you can migrate existing on-premise or cloud-hosted PostgreSQL databases to clusters in your Aiven account.
+
+  .. note:: 
+    
+     The console migration tool doesn't support migrating managed database clusters on Aiven to other managed database clusters on Aiven.
```

Comment:
I need to check this still, it should be possible actually with `aiven_extras` installed.

Reply:
I've checked between the Aiven services and it worked, using avnadmin user and after installing `aiven_extras` on the source.

Reply:
@alexole , thank you - removing the note in that case :-)